cherry-core.php

```diff
@@ -286,7 +286,7 @@ public function get_module_instance( $module, $args = array() ) {
 			$class_name = self::get_class_name( $module );
 
 			if ( ! class_exists( $class_name ) ) {
-				echo '<p>Class <b>' . $class_name . '</b> not exist!</p>';
+				echo '<p>Class <b>' . esc_attr( $class_name ) . '</b> not exist!</p>';
```

Comment:
Use `esc_html` instead of `esc_attr`.
esc_attr - for escaping for HTML **attributes**.